src/CommandingOfficers/CommanderPatch.java

```diff
@@ -13,8 +13,24 @@
   private static final long serialVersionUID = -543799935457360451L;
   
   private static final CommanderInfo coInfo = new CommanderInfo("Patch", new instantiator());  
-  private static class instantiator implements COMaker
+  private static class instantiator extends COMaker
   {
+    public instantiator()
+    {
+      infoPages.add(new InfoPage(
+          "Commander Patch is a pirate, who does piratey things like lootin' and plunderin'\n"));
+      infoPages.add(new InfoPage(
+          "Passive:\n" + 
+          "- Patch gets a turn's worth of income from any property he captures\n"));
+      infoPages.add(new InfoPage(
+          "Plunder ("+PLUNDER_COST+"):\n" + 
+          "Gives an attack boost of "+PLUNDER_ATTACK_BUFF+"%\n" + 
+          "Gives "+PLUNDER_INCOME+"x of the value of any funds damage Patch deals.\n"));
+      infoPages.add(new InfoPage(
+          "Plunder ("+PILLAGE_COST+"):\n" + 
+          "Gives an attack boost of "+PILLAGE_ATTACK_BUFF+"%\n" + 
+          "Gives "+PILLAGE_INCOME+"x the value of any funds damage Patch deals.\n"));
+    }
```

Comment:
We should probably tweak this guy's numbers. Costs of 5 and 10 seem a bit steep for his abilities.

Reply:
As part of this PR?

Reply:
Nah. Out of scope.